.github/README.md

```diff
@@ -1,32 +1,31 @@
 # Q-CTRL DevOps Challenge
 
-The Q-CTRL DevOps Challenge is a way for applicants applying for a role within the Q-CTRL DevOps Engineering team to demonstrate their overall approach to creating solutions from a variety of requirements.
+The Q-CTRL DevOps Challenge is a way for applicants to roles within the Q-CTRL DevOps Engineering team to demonstrate their skills and overall approach to solving a challenge based upon a set of requirements.
+
+The requirements are intentionally sparse (the devil is in the details). We don't want you to do a lot, but what you do should be your best work and should clearly demonstrate you embody [the three virtues](https://thethreevirtues.com/).
 
 ## The rules
 
-- Read and understand [The challenge](#the-challenge).
-- Create a solution that satisfies the [Requirements](#requirements).
-- Send the link to the repository containing your solution to the person who contacted you.
+- Read and understand [the challenge](#the-challenge).
+- Create a solution that satisfies [the requirements](#the-requirements).
+- Send a link to the repository containing your solution to the person who contacted you.
+
+**IMPORTANT:** The repository containing your solution must be public, must contain a `README.md` file, **MUST NOT** be a fork of this repository, and **MUST NOT** make any reference to "Q-CTRL" in the repository name, description, or code.
+
+**Note:** You should spend no more than three hours on this challenge.
 
 ## The challenge
```

Comment:
@cfarrend see https://github.com/qctrl/front-end-challenge#the-challenge for a good challenge.